azurerm/resource_arm_container_group.go

```diff
@@ -0,0 +1,294 @@
+package azurerm
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/Azure/azure-sdk-for-go/arm/containerinstance"
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/hashicorp/terraform/helper/validation"
+)
+
+func resourceArmContainerGroup() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceArmContainerGroupCreate,
+		Read:   resourceArmContainerGroupRead,
+		Update: resourceArmContainerGroupCreate,
+		Delete: resourceArmContainerGroupDelete,
+
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:     schema.TypeString,
+				Required: true,
```

Comment:
I think this also needs to be `ForceNew`?

Reply:
Done.